src/shogun/CMakeLists.txt

```diff
@@ -217,6 +217,21 @@ CHECK_FUNCTION_EXISTS(lgammal HAVE_LGAMMAL)
 
 ############################ external dependencies
 
+#Stan math
+include(external/StanMath)
+SHOGUN_INCLUDE_DIRS(SCOPE PUBLIC SYSTEM
+  $<BUILD_INTERFACE:${STAN_INCLUDE_DIR_STAN_MATH}>
+  $<INSTALL_INTERFACE:include/shogun/lib/external/stan1>
```

Comment:
what's the `stan1` `stan2` `stan3` stands for?

Reply:
Changed the names to be more descriptive.